enumerable.rb

```diff
@@ -0,0 +1,26 @@
+module MyEnumerable
+  # filter method
+  def filter(&block)
+    ans = []
+    each do |items|
+      result << items if block.call(items) == true
+    end
+    ans
```

Comment:
- [ ] when I run the filter methods it returns an error in line 6 it should be `ans` not `result ` because result is not defined so please try to rename array ans to result in order to complete the requiements